### ISSUE TITLE
option to disable bottom menu

### ISSUE THREAD

**Issue by ilf** [FredJul/Flym#474]
IMHO, the bottom menu (Unreads, All, Favorites) should be optional.
It takes up quite some screen space, not everyone uses it, and it can be an option in the top main menu.

**Comment by ildar** [comment]
Optional autohide should be sufficient (?)

**Comment by ilf** [comment]
> Optional autohide should be sufficient (?)

Yes, that sounds exactly like what I am proposing. I have adjusted the title.

Or do you mean something different?
